test-cargo-miri/run-test.py

```diff
@@ -90,6 +90,12 @@ def test_no_rebuild(name, cmd, env={}):
         fail("Something was being rebuilt when it should not be (or we got no output)");
 
 def test_cargo_miri_run():
+    # Try to remove cargo config to avoid unexpected settings
+    try:
+        os.remove('.cargo/config.toml')
```

Comment:
I just realized this seems to edit the same file as https://github.com/rust-lang/miri/blob/aa53f3f77fea74e5a99e700e24b620cb5ad8242a/ci.sh#L49...

Reply:
Oh yeah, the path should be absolute here

Reply:
That's not what I mean. It can be relative, the other paths in the python file are also relative.

What I mean is that removing that file removes testing the `build.rustc-wrapper = "thisdoesnotexist"` thing!